runtime/wasm/src/host_exports.rs

```diff
@@ -150,22 +150,6 @@ impl<C: Blockchain> HostExports<C> {
         );
         poi_section.end();
 
-        let id_insert_section = stopwatch.start_section("host_export_store_set__insert_id");
```

Comment:
Shouldn't we keep this to measure the performance with `Bytes`?

Reply:
This was only measuring the time it took to insert into an in-memory map, so not super-interesting; I moved that logic to `EntityCache.set` and this operation now gets measured as part of the `host_export_store_set` section which I have never seen take interesting amounts of time.

Reply:
Oh okay, makes sense!